primitives/src/sequence.rs

```diff
@@ -25,7 +25,7 @@ use units::parse::{self, PrefixedHexError, UnprefixedHexError};
 
 use crate::locktime::relative;
 #[cfg(all(doc, feature = "alloc"))]
-use crate::transaction::Transaction;
+use crate::{CompactTarget, Transaction};
```

Comment:
In 6e0deb9f6b2dc6766bba02c3601ea75d4586fa6b:

This looks unnecessary. I think it's stale from when you had mistakenly added [`CompactTarget`] to the docstring.

Reply:
Damn, it is hard to get good help.